package.json

```diff
@@ -5,7 +5,7 @@
   "description": "Multivalue Object Mapper",
   "main": "index.js",
   "engines": {
-    "node": ">=14.19"
+    "node": ">=16.19"
```

Comment:
I'm honestly not sure of best practices here.  Should we support node starting at 16.0 or is it okay to make the range a high patch range?

Presumably we currently work alright with 16.0 and higher since it was in the earlier semver range.  I would think we'd want to keep this band wider unless we know of something that will not work with an earlier version.